### ISSUE TITLE
Import in module fails due to default export

### ISSUE THREAD

**Issue by F-Node-Karlsruhe** [transmute-industries/vc-jwt-sd#4]
When doing `import SD from '@transmute/vc-jwt-sd'` the default export returns an object like so `{default: SD}`

the current import into an ES2022 module can only be done like so:

```
import SD from '@transmute/vc-jwt-sd';
const vcjwtsd = (SD as any).default;
```

**Comment by OR13** [comment]
Yes, thank you for pointing this out, the next version will not have default export at the top level.
